### ISSUE TITLE
Error when opening quick switcher

### ISSUE THREAD

**Issue by HEmile** [darlal/obsidian-switcher-plus#12]
Hi,
When opening the quick switcher, Im  getting this error, and the switcher doesn't suggest any options.

```
VM157:450 Uncaught TypeError: Cannot read property '0' of null
    at ModeHandler.onInput (eval at evalFunc (app.js:1), <anonymous>:450:47)
    at class_1.onInput (eval at evalFunc (app.js:1), <anonymous>:849:20)
    at class_1.onOpen (eval at evalFunc (app.js:1), <anonymous>:841:18)
    at class_1.e.open (app.js:1)
    at class_1.openInMode (eval at evalFunc (app.js:1), <anonymous>:834:18)
    at Object.checkCallback (eval at evalFunc (app.js:1), <anonymous>:922:31)
    at dt (app.js:1)
    at e.executeCommandById (app.js:1)
    at e.onTrigger (app.js:1)
    at e.handleKey (app.js:1)
``` 

I'm fairly certain it's related to this plugin, as the core switcher didn't have this problem

**Comment by pcuellar** [comment]
I am getting the same behavior with a similar error.
``` 
9app.js:1 Uncaught TypeError: Cannot read property 'options' of undefined
    at class_1.t.getSuggestions (app.js:1)
    at class_1.t.updateSuggestions (app.js:1)
    at class_1.updateSuggestions (eval at <anonymous> (app.js:1), <anonymous>:855:52)
    at class_1.t.onInput (app.js:1)
    at class_1.onInput (eval at <anonymous> (app.js:1), <anonymous>:850:38)
```
Tested with no theme and it still doesn't work. Same as @HEmile, I am getting suggestions under the core switcher functionality.

**Comment by darlal** [comment]
These are actually 2 different issues. 

@HEmile, I have not been able to reproduce the error you reported. If you're still having this issue, please provide some more details around step to reproduce. Either way, there is some sensible error handling that can be added to that part of the code.

@pcuellar, the one that you are seeing is the result of the recent 12.4 core obsidian insider release changes. We'll get an update out for this shortly.

**Comment by darlal** [comment]
Fixed in [0.11](https://github.com/darlal/obsidian-switcher-plus/releases/tag/0.0.11) release. Please upgrade when you get a chance. Thanks! 

We'll close this issue out.
